publish/models/dandiset.py

```diff
@@ -30,7 +30,7 @@ class Meta:
 
     @property
     def identifier(self) -> Optional[str]:
-        return f'{self.id:06}' if self.id else ''
+        return f'{self.id:06}' if self.id is not None else ''
```

Comment:
```suggestion
        return f'{self.id:06}' if self.id is not None else '' # Guard the case of identifier 0
```

Reply:
That comment only really makes sense in the context of the diff here. Would it make more sense to generally describe the function? Something like

```# Return the zero-padded id if it exists.```

Reply:
`# Return the zero-padded id if it exists.` describes what the code does (which any developer can already understand), not *why*. It's fine for a docstring, but unhelpful as a code comment.

If we feel a comment is necessary, perhaps just be explicit: `# Compare against None, to allow id 0`.

Reply:
Also, it's our style convention to have code comments always occupy their own line, unless it's a `# noqa: `.